### ISSUE TITLE
Support Django 2.0 and 2.1

### ISSUE THREAD

**Issue by uri-rodberg** [deschler/django-modeltranslation#472]
When will you release a stable (not beta) release supporting Django 2.0? django-modeltranslation is one of only 2 packages in our project (https://github.com/speedy-net/speedy-net) not supporting Django 2.0 with stable releases (the other one is django-friendship, which released a version supporting Django 2.0, but it has bugs/issues).

**Comment by urievenchen** [comment]
Currently django-modeltranslation is the only package in our project not supporting Django 2.0 with a stable release (django-friendship released a new version supporting Django 2.0 without known bugs/issues).

**Comment by last-partizan** [comment]
Any news on this?

**Comment by karolyi** [comment]
refers #467

**Comment by lifenautjoe** [comment]
Tried installing the beta. Admin does not work. 

````bash
pipenv install -e git+https://github.com/deschler/django-modeltranslation.git@0.13-beta1#egg=django-modeltranslation
```

**Comment by ulgens** [comment]
@deschler Are you okay? :slightly_smiling_face:

**Comment by ulgens** [comment]
@deschler Is everything okay there? :slightly_smiling_face:

**Comment by uri-rodberg** [comment]
Django 2.0 was released on December 2017, and Django 2.1 on August 2018, so I think it's about time for django-modeltranslation to support them.

**Comment by lifenautjoe** [comment]
Hi @deschler , everything alright there mate? 🙏

**Comment by gabn88** [comment]
Hi @deschler, hope you are alright and thanks for creating this fine package. We have used it for several years without any real issues. Many thanks for that.

Due to lack of maintenance of this package and the need to upgrade to Django 2.0+, and use of features that this package does not support (such as certain queries), I have upgraded our staging server to django-translated-fields. I am not familiar with the author, but the package is only 50 lines and therefore very understandable. It is also very basic, but with some glue code ([for example like I wrote down here here](https://github.com/matthiask/django-translated-fields/issues/7#issuecomment-457340307), we were able to make the switch fairly quick. Added bonus is that all advanced queries are working now. We are still testing, but it looks promising!
